examples/tv-app/linux/include/media-playback/MediaPlaybackManager.cpp

```diff
@@ -125,81 +135,84 @@ EmberAfMediaPlaybackStatus MediaPlaybackManager::proxyMediaPlaybackRequest(Media
+    sendResponse(command, "MediaPrevious", ZCL_MEDIA_PREVIOUS_RESPONSE_COMMAND_ID, status);
     return true;
 }
 
-bool emberAfMediaPlaybackClusterMediaRewindCallback(chip::app::Command *)
+bool emberAfMediaPlaybackClusterMediaRewindCallback(chip::app::Command * command)
 {
-    EmberAfMediaPlaybackStatus status = MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::Rewind);
-    sendResponse("MediaRewind", ZCL_MEDIA_REWIND_RESPONSE_COMMAND_ID, status);
+    EmberAfMediaPlaybackStatus status = MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::Rewind, 0);
+    sendResponse(command, "MediaRewind", ZCL_MEDIA_REWIND_RESPONSE_COMMAND_ID, status);
     return true;
 }
 
-bool emberAfMediaPlaybackClusterMediaSkipBackwardCallback(chip::app::Command *, uint64_t deltaPositionMilliseconds)
+bool emberAfMediaPlaybackClusterMediaSkipBackwardCallback(chip::app::Command * command, uint64_t deltaPositionMilliseconds)
 {
-    EmberAfMediaPlaybackStatus status = MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::SkipBackward);
-    sendResponse("MediaSkipBackward", ZCL_MEDIA_SKIP_BACKWARD_RESPONSE_COMMAND_ID, status);
+    EmberAfMediaPlaybackStatus status =
+        MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::SkipBackward, deltaPositionMilliseconds);
+    sendResponse(command, "MediaSkipBackward", ZCL_MEDIA_SKIP_BACKWARD_RESPONSE_COMMAND_ID, status);
     return true;
 }
 
-bool emberAfMediaPlaybackClusterMediaSkipForwardCallback(chip::app::Command *, uint64_t deltaPositionMilliseconds)
+bool emberAfMediaPlaybackClusterMediaSkipForwardCallback(chip::app::Command * command, uint64_t deltaPositionMilliseconds)
 {
-    EmberAfMediaPlaybackStatus status = MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::SkipForward);
-    sendResponse("MediaSkipForward", ZCL_MEDIA_SKIP_FORWARD_RESPONSE_COMMAND_ID, status);
+    EmberAfMediaPlaybackStatus status =
+        MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::SkipForward, deltaPositionMilliseconds);
+    sendResponse(command, "MediaSkipForward", ZCL_MEDIA_SKIP_FORWARD_RESPONSE_COMMAND_ID, status);
     return true;
 }
 
-bool emberAfMediaPlaybackClusterMediaSkipSeekCallback(chip::app::Command *, uint64_t positionMilliseconds)
+bool emberAfMediaPlaybackClusterMediaSkipSeekCallback(chip::app::Command * command, uint64_t positionMilliseconds)
 {
-    EmberAfMediaPlaybackStatus status = MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::Seek);
-    sendResponse("MediaSeek", ZCL_MEDIA_SKIP_FORWARD_RESPONSE_COMMAND_ID, status);
+    EmberAfMediaPlaybackStatus status =
+        MediaPlaybackManager().proxyMediaPlaybackRequest(MediaPlaybackRequest::Seek, positionMilliseconds);
+    sendResponse(command, "MediaSeek", ZCL_MEDIA_SKIP_FORWARD_RESPONSE_COMMAND_ID, status);
```

Comment:
This doesn't look like the right response command id?  Should be ZCL_MEDIA_SKIP_SEEK_RESPONSE_COMMAND_ID?  But also, these commands and the XML look nothing like the spec, so I'm a little confused by that....

Reply:
Updated. Now seek returns the `ZCL_MEDIA_SKIP_SEEK_RESPONSE_COMMAND_ID`. 

I don't get the comment regarding [the spec](https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/media/MediaPlayback.adoc#seek-request-command).

Seems up to date?

Reply:
So for example.  The XML has a `MediaSkipSeek` command defined as going from client to server, with command ID 0x0A and a `MediaSkipBackwardResponse` command defined as going server to client with command ID 0x09.

In the spec linked right there the only command with ID 0x09 is `SkipBackward Request`, going client to server, and the only command with `0x0A` with `Playback Response`, going server to client.

As I said, "looks nothing like the spec"...

Reply:
Updated. Renamed the command from SkipSeek to just Seek and updated id to 0x0B. 

Regarding the response commands. You cannot have one shared response command for in this case 10 commands. You have to have a custom made 10 commands for each  of 10 commands.

Reply:
> You cannot have one shared response command for in this case 10 commands. 

Why not?  I mean, lots of commands just use Default Response as the response....

In general, the mapping from commands to responses is many-to-one, as far as I can tell.

More to the point: Is this "cannot" a claim about ZAP, or our codebase, or the spec?  I _think_ it's not accurate for any of those, but if I'm wrong I would really like to know where I'm wrong.